pkg/logs/client/http/destination_test.go

```diff
@@ -17,9 +18,16 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/logs/message"
 
 	"github.com/DataDog/datadog-agent/comp/logs/agent/config"
+	pkgconfigmodel "github.com/DataDog/datadog-agent/pkg/config/model"
 	"github.com/DataDog/datadog-agent/pkg/util/pointer"
 )
 
+var cfg pkgconfigmodel.Reader
+
+func init() {
+	cfg = pkgconfigmodel.NewConfig("test", "DD", strings.NewReplacer(".", "_"))
```

Comment:
We should avoid adding global state like this. We are in the process of trying to remove `init()` hooks from the agent.

Perhaps you can create a helper function that returns a new config that each test can use?